charts/activemq/templates/_helpers.tpl

```diff
@@ -48,6 +48,7 @@ Selector labels
 {{- define "activemq.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "activemq.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/component: {{ template "activemq.name" . }}
```

Comment:
I think we should avoid using any `.name` named template, if you check the current implementation you see that is namespaced with the helm release name, while I imagine we want a fixed string which identity the component name (multiple helm install should yield the same component name).

I guess using `{{ .Chart.Name }}` should work